pkgs/tools/networking/babeld/default.nix

```diff
@@ -1,11 +1,11 @@
 { stdenv, fetchurl }:
 
 stdenv.mkDerivation rec {
-  name = "babeld-1.9.0";
+  name = "babeld-1.9.1";
```

Comment:
would like to see `pname` and `version` be used, but it's a backport, so I think this is alright. Just wanted to make a note, but don't change it, unless someone else feels more strongly about it :)

Reply:
Do you have a link where I can read up on the `pname` thing?

Reply:
it's for the automatic updates using repology.
Since this is a backport to a stable branch, I feel that it should reflect what is in master, otherwise here is the main link:
https://github.com/NixOS/nixpkgs/issues/61691